.github/workflows/bokeh-ci.yml

```diff
@@ -12,8 +12,8 @@ defaults:
     shell: bash -l {0}
 
 env:
-  CHROME_VER: "110.0.5481.100"
-  CHROME_REV: "chromium_2333"
+  CHROME_VER: '110.0.5481.100'
+  CHROME_REV: 'chromium_2333'
```

Comment:
What's the rationale for this and other changes like that?

Reply:
Well, these changes were done by saving and my editor doing its own thing. I can revert that change of course!

Reply:
@DerTimonius please do revert them